src/extensions/UserDefinedFormControllerExtension.php

```diff
@@ -2,19 +2,39 @@
 
 namespace Firesphere\PartialUserforms\Extensions;
 
+use Firesphere\PartialUserforms\Models\PartialFormSubmission;
 use SilverStripe\Core\Extension;
 use SilverStripe\UserForms\Control\UserDefinedFormController;
 use SilverStripe\View\Requirements;
 
 /**
  * Class UserDefinedFormControllerExtension
+ *
  * @package Firesphere\PartialUserforms\Extensions
  * @property UserDefinedFormController|UserDefinedFormControllerExtension $owner
  */
 class UserDefinedFormControllerExtension extends Extension
 {
+    private static $allowed_actions = [
+        'verify'
+    ];
+    /**
+     * @var null|PartialFormSubmission
+     */
+    protected $partialForm;
+
     public function onBeforeInit()
     {
         Requirements::javascript('firesphere/partialuserforms:client/dist/main.js');
     }
+
+    public function verify()
+    {
+        return $this;
+    }
+
+    public function getPartialForm()
```

Comment:
I'm assuming you're planning to use this in the future?

Reply:
As above 🤦‍♂